src/action.rs

```diff
@@ -6,7 +6,7 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
-use crate::{authority::Authority, error::InterfaceError, xor_space::XorName};
```

Comment:
commit titles could be a bit more helpful, than just the scope and type of change.
We were also following the leaf guideline https://github.com/autumnai/leaf/blob/master/CONTRIBUTING.md#git-commit-guidelines so all lower case type/scope:

`refactor/routing: renamed Authority to Location`
`docs/routing: updated documentation for Location struct`
`docs/routing: rename authorities -> locations in docs.`

But in the case of this change, might as well keep a single commit (squash and fix the single commit) will make it easier to rebase as well.